src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAOSpec.scala

```diff
@@ -33,6 +33,27 @@ class GraphEntityDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture
     }
   }
 
+  it should "correctly count entities of a certain type" in withDefaultTestDatabase { dataSource =>
+    dataSource.inTransaction(readLocks=Set(testData.workspace.toWorkspaceName)) { txn =>
+      withWorkspaceContext(testData.workspace, txn) { context =>
+        val bernie = Entity("Bernie", "Politician", Map.empty)
+        val obama = Entity("Obama", "Politician", Map.empty)
+        val biden = Entity("Biden", "Politician", Map.empty)
+        val trump = Entity("Trump", "Politician", Map.empty)
+
+        entityDAO.save(context, bernie, txn)
+        entityDAO.save(context, obama, txn)
+        entityDAO.save(context, biden, txn)
+        entityDAO.save(context, trump, txn)
+
+        assertResult(4) {
+          dao.getEntityTypeCount(context, "Politician", txn)
+        }
+
+      }
+    }
+  }
+
```

Comment:
I changed ```entityPipeline``` to use ```entityTypePipeline``` so it will be checked in all of the old unit tests that listed entities for a certain type